frontend/src/lib/components/Card.svelte

```diff
@@ -1,7 +1,11 @@
 <script>
-    let clazz = undefined
-    export { clazz as class }
+	let clazz = undefined;
+	export { clazz as class };
 </script>
-<div class="p-4 shadow border-l-4 border-feather-light {clazz || ""}">
-    <slot></slot>
-</div>
\ No newline at end of file
+
+<div
+	class="p-6 shadow rounded-md bg-gray-100 dark:bg-gray-800 transition-colors duration-200 {clazz ||
```

Comment:
`$$restProps` or `$$props`

Reply:
what do you mean by that?

Reply:
Reference to `clazz` I'm not sure if we are allowed to have more than one `class` though.

Reply:
Also, it might be me who made this...